src/services/pdf-generation.ts

```diff
@@ -40,13 +41,14 @@ export default class PdfGenerationService {
     }
 
     async launchBrowser(pdfGenerationRequest: PdfGenerationRequest) {
-        return await chromium.puppeteer.launch({
+        const chromiumPath = await chromium.executablePath();
+        const options = {
             args: chromium.args,
-            defaultViewport: null,
-            executablePath: await chromium.executablePath,
-            headless: chromium.headless,
-            ignoreHTTPSErrors: true,
+            executablePath: chromiumPath,
+            headless: !!process?.env?.BROWSER_HEADLESS === false ? false : true,
```

Comment:
Probably should follow what the spazium lib does: https://github.com/Sparticuz/chromium?tab=readme-ov-file#usage and use `chromium.headless`